mavros_extras/src/plugins/odom.cpp

```diff
@@ -0,0 +1,184 @@
+/**
+ * @brief Odometry plugin
+ * @file odom.cpp
+ * @author James Goppert <james.goppert8@gmail.com>
```

Comment:
Mail fixed. No need to add my email. It was your idea and your implementation. I just added some features and robustness ;)